src/DotNetWorker.Core/Hosting/WorkerOptions.cs

```diff
@@ -21,5 +21,10 @@ public class WorkerOptions
         /// Gets the collection of input converters.
         /// </summary>
         public InputConverterCollection InputConverters { get; } = new InputConverterCollection();
+        
+        /// <summary>
+        /// Gets or sets a value that determines if empty entries should be included in the function trigger message payload.
```

Comment:
It would be good to have more details here to make the behavior controlled by the option clearer, perhaps also outline the current behavior?

Reply:
@fabiocav I tried to elaborate the scenario a bit more in the latest iteration. Could you take a look? Feel free to suggest a better version if we can improve this further. Thank you!